cmd/snap/cmd_snapshot_test.go

```diff
@@ -156,7 +156,7 @@ func (s *SnapSuite) TestSnapshotImportHappy(c *C) {
 	expectedAge := time.Since(time.Now().AddDate(0, -1, 0))
 	ageStr := quantity.FormatDuration(expectedAge.Seconds())
 	// 30d0h (no DST change), 30d1h (summer to winter time), 29d23h (winter to summer time)
-	c.Check(ageStr, Matches, `(30d0h|30d1h|29d23h)`)
+	c.Check(ageStr, Matches, `(31d0h|31d1h|30d0h|30d1h|29d23h)`)
```

Comment:
This will see us through until March next year (since February has neither 30 or 31 days).  Is this check actually useful?

Reply:
Yeah, too much hassle. Let's drop this check.